modules/canary/plans/init.pp

```diff
@@ -1,3 +1,30 @@
+# @summary
+#   Run a task, command or script on canary nodes before running it on all nodes.
+#
+# This plan accepts a task/command/script and a $nodes parameter. It will run
+# the task on a canary group of of nodes and only continue to the rest of the
+# nodes if it succeeds on all canaries. This returns a ResultSet object with a
+# Result for every node. Any skipped nodes will have a 'canary/skipped-node'
+# error kind.
+#
+# @param task
+#  The name of the task to run. Mutually exclusive with command and script.
+# @param command
+#   The command to run. Mutually exclusive with task and script.
+# @param script
+#   The script to run. Mutually exclusive with task and command.
+# @param nodes
+#   The target to run on.
+# @param params
+#   The parameters to use for the task.
```

Comment:
```suggestion
#   The parameters to use for the action.
```

Reply:
Users should not generally set parameters for non-task actions.